lib/ts/recipe/oauth2provider/index.ts

```diff
@@ -107,6 +107,28 @@ export default class Wrapper {
             userContext: getUserContext(userContext),
         });
     }
+
+    static revokeToken(
+        token: string,
+        clientId: string,
+        clientSecret: string,
+        useBasicAuth = false,
```

Comment:
What happens if the use passes the wrong value here? How does this interact with clients that do not have clientSecrets?

Reply:
We fixed it by fetching the clientInfo and passing the values correctly.